### ISSUE TITLE
Suggestion - Default Weightings

### ISSUE THREAD

**Issue by wasniahC** [xScone/Gnomes-Dev#5]
Most enemies have a spawn weight of between 20 and 60 (sometimes pushing around 80 for things like brackens on forest worlds, or snare fleas on assurance)

Currently the default spawn weights on gnomes are 100 and 300. This weighting means they have a roughly 30-60% spawn rate on most maps, and on smaller moons this crowds out the 3-power enemies from spawning (so barely seeing brackens on vow). I've found around 30 is a more sensible default weight for either/both gnomes and would suggest tinkering with your defaults - current defaults don't feel like they integrate with the game so much as just turn it into Gnomes Company.

**Comment by allvei** [comment]
Agreed, please reduce gnome spawnrate, takes out the fun having to deal with 3-5 gnomes at once.
